Rext/ProxyHandler.cs

```diff
@@ -12,14 +12,21 @@ public static HttpClientHandler ProxyHandler(string address = null, bool relaxSs
         {
             HttpClientHandler proxyHandler = new HttpClientHandler();
 
-            if (!string.IsNullOrEmpty(address))
+            if (Uri.IsWellFormedUriString(address, UriKind.Absolute))
```

Comment:
Please test with a null proxy and see if exception is thrown. If the proxy value is null then it should not crash the process

Reply:
Yea... Am fixing that right away